Server/models/Metrics.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require("mongoose");
+
+/**
+ * Mongoose schema for storing metrics from Google Lighthouse.
+ * @typedef {Object} MetricsSchema
+ * @property {mongoose.Schema.Types.ObjectId} monitorId - Reference to the Monitor model.
+ * @property {Object} accessibility - Metrics related to accessibility category.
+ * @property {Object} bestPractices - Metrics related to best practices category.
+ * @property {Object} seo - Metrics related to SEO category.
+ * @property {Object} performance - Metrics related to performance category.
+ * @property {Date} timestamp - Timestamp when the metrics were recorded.
+ */
+
+const MetricsSchema = mongoose.Schema(
```

Comment:
Why don't we name this PageSpeedCheck as that describes exactly what this object is